2020-03-Autumn/0_sliding_window/README.md

```diff
@@ -94,21 +105,62 @@ Go to [Solution](#s3)   [Top](#top)
 <!-- Don't remove -->
 <a name="s1"/>
 
-### 1. SOLUTION 1 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution
-
-TODO :bug:
+### 1. Max Consecutive Ones
+
+Source: [LeetCode](https://leetcode.com/problems/max-consecutive-ones/)
+
+#### Brute Force Solution
+
+At the first glance of this question, we can see that the program wants to 
+count the most 1s that are grouped together. As we go through the binary 
+array, we can count along the way: If it is a 0, we would stop counting 
+and reset the counter to 0. Otherwise, we would increment the counter by 1. 
+The maximum number of the counter will always be stored in a variable called 
+`max`.
+
```

Comment:
This isn't quite the brute force solution. I'd actually say it's another way of solving this problem optimally in `O(N)` time without a sliding window.

If you can, I'd try to add the `O(N^2)` brute force solution - something like this:

```
max_count = 0
n = len(arr)
for i in range(n):
 if i == 1:
    count = 0
    for j in range(i, n):
        if arr[j] == 1:
           count += 1
        else:
            max_count = max(count, max_count)
            break
return max_count
```

Reply:
Changes addressed in commit 8a11068.